src/status_im/constants.cljs

```diff
@@ -446,6 +446,8 @@
 (def ^:const optimism-short-name "opt")
 (def ^:const arbitrum-short-name "arb1")
 
+(def ^:const default-address-prefix "eth:opt:arb1:")
```

Comment:
so in Status Go this is referred to as:
:prod-preferred-chain-ids 
and in the design it's generally referred to as 
"network-preferences".

I think we should be consistent in our language for concepts across the codebase.

Imo address-prefix does not suggest the use case and so I prefer the use of `default-network-preference` or 
`default-network-preferences-prefix`.

cc @OmarBasem, @shivekkhurana, @smohamedjavid as I think we were having the same discussion in another pr.
I don't mind which we choose but let's at least be consistent across usages 👍

Reply:
Agree, I don't have a strong opinion on this, I can change it to whatever name we agree on!

Reply:
@J-Son89 I don't mind using another name, but we need to differentiate but the current user's network preferences, and the "address prefix" of a receiver when sending tokens

Reply:
Both `network-preferences` and `address-prefix` are valid names here. The former from the backend perspective, and latter from UI perspective. 

But I see this leaking in multiple PR. I have introduced these methods to encapsulate them:

```clj
(defn chain-ids->address-prefix [chain-ids])

(defn chain-short-ids->address-prefix [chain-short-ids])

(defn address-prefix->chain-ids [adress-prefix])

(defn address-prefix->chain-short-ids [address-prefix])```

There will probably be a few more to manage all use-cases.